test/util/testfactory/utils.go

```diff
@@ -346,14 +289,15 @@ func GetAccountSpendableBalance(grpcConn *grpc.ClientConn, address string) (bala
 	return res.GetBalances(), nil
 }
 
-// GetAccountSpendableBalanceByBlock retrieves the spendable balance of an account for the specified address at the given height using gRPC.
-// It takes a gRPC client connection (grpcConn) and the account address (address) as inputs.
-// If the account is not found or an error occurs, it returns nil and the error.
-// Otherwise, it returns the spendable balances of the account as an sdk.Coins object.
+// GetAccountSpendableBalanceByBlock retrieves the spendable balance of an account for the specified address at the given block using gRPC.
 func GetAccountSpendableBalanceByBlock(grpcConn *grpc.ClientConn, address string, block *coretypes.Block) (balances sdk.Coins, err error) {
 	cli := banktypes.NewQueryClient(grpcConn)
 	ctx := metadata.AppendToOutgoingContext(context.Background(), grpctypes.GRPCBlockHeightHeader, fmt.Sprint(block.Height))
-	ctx = metadata.AppendToOutgoingContext(ctx, grpctypes.GRPCBlockTimeHeader, fmt.Sprint(block.Time.Unix()))
+
+	// Since the blockTime is not inferred from block height to the GRPC server, we use the following
+	// line; however, it is commented out because we do not want to modify the SDK code to support it
+	// ctx = metadata.AppendToOutgoingContext(ctx, grpctypes.GRPCBlockTimeHeader, fmt.Sprint(block.Time.Unix()))
```

Comment:
[blocking] I don't understand this. Why do we want to merge the commented code?

Reply:
do you think we should remove that function entirely as it is not used in the current test code?

Reply:
removed